### PR TITLE
Vertical scrollbars meta-PR

### PR DESCRIPTION
**Pull request by bcjordan**
This contains many already-reviewed PRs:
#3005
#3007
#3050
#3051
#3054
#3056
#3077
#3079
#3118
#3117
#3201

Now looking at enabling for a subset of apps and testing thoroughly.

### CODE REVIEW

File: dashboard/app/models/game.rb [hunk 1]
```diff
@@ -92,6 +92,10 @@ def uses_small_footer?
     app == NETSIM
   end
 
+  def uses_small_footer?
```
Comment (bcjordan) [inline dashboard/app/models/game.rb]: Issue from auto-merge, resolved. :+1: woo hound

**Comment by bcjordan** [comment]
@islemaster I've merged with latest staging contents and am doing a manual test pass locally now.

Can you give a once-over and holler if you see anything obvious? Specifically anything around the small footer merge or enabling/disabling logic.

The intent is for this branch's merging to not have any user-facing effects, i.e., it shouldn't enable vertical scrollbars or footer pinning anywhere it isn't yet.

File: apps/src/StudioApp.js [hunk 2]
```diff
@@ -645,6 +645,7 @@ StudioApp.prototype.localeDirection = function() {
 * Blockly.Xml.domToText(Blockly.Xml.blockSpaceToDom(Blockly.mainBlockSpace)).slice(5, -6)
 */
 StudioApp.prototype.initReadonly = function(options) {
+  // TODO (bbuchanan): Change readonly scrolling rules here, maybe inherit from level
```
Comment (islemaster) [inline apps/src/StudioApp.js]: Should maybe address or remove this TODO before merge.

**Comment by islemaster** [comment]
Did a quick scan, nothing alarming.  Looks like this is configured correctly to be disabled everywhere for now.  There will be some additional work (a different PR, I hope!) to enable the small footer in AppLab.

Note, there are user-facing effects to this PR but they should all be bugfixes :grin:.

**Pull request merged**